### PR TITLE
Append multiple screenshot in issue json

### LINKED ISSUE #1075: Update GitHub post code to work with multiple screenshots
EA70146B-1262-43A8-8EC3-BB27D7E660D3

move it below the screenshots processing loop and append each file to the issue json

### PR DESCRIPTION
**Pull request by Kej-r03**
Fixes #1075

### CODE REVIEW

**Review by fredfalcon — CHANGES_REQUESTED** [review]
This works however I think it will be good to have all screenshots attached. Thanks

**Comment by Kej-r03** [comment]
I have attached a screen recording of the working. You can see that we do not get any console errors. Thanks

https://user-images.githubusercontent.com/99071926/220536801-4e4dedde-349b-42c8-8a0d-ef427b29ce47.mp4

**Review by fredfalcon — CHANGES_REQUESTED** [review]
Can you please make it attach all screenshots to GitHub?

**Review by fredfalcon — CHANGES_REQUESTED** [review]
You should be able to use the IssueScreenshot.image.url function to get the url - Django has a built in image url function.

**Comment by Kej-r03** [comment]
Ok, I think the first commit was better itself, so I have reverted back. I have created an issue using the github api post method [here](https://github.com/Kej-r03/Practice3/issues/15). Here too the image is not fully loaded though

**Review by fredfalcon — CHANGES_REQUESTED** [review]
add string to first loop

File: website/views.py [hunk 1]
```diff
@@ -676,13 +676,30 @@ def form_valid(self, form):
...
+            filename = screenshot.name
+            extension = filename.split(".")[-1] 
+            screenshot.name = filename[:99] + str(uuid.uuid4()) + "." + extension            
+            default_storage.save(f"screenshots/{screenshot.name}",screenshot)
+            IssueScreenshot.objects.create(image=f"screenshots/{screenshot.name}",issue=obj)
+
+        obj_screenshots = IssueScreenshot.objects.filter(issue_id=obj.id)
+        screenshot_text = ''
+        for screenshot in obj_screenshots:
+            screenshot_text += "![0](" + screenshot.image.url + ") "
```
[the remark this answers is not in the document]
Comment (Kej-r03) [reply website/views.py:694]: Yes, I have tried this and this gave an error 'ImageFileDescriptor' object has no attribute 'url'. [my previous comment on this](https://github.com/OWASP/BLT/issues/1075#issuecomment-1439479230)

**Review by fredfalcon — APPROVED** [review]

**Pull request merged**